core/modules/admin_bar/css/admin_bar.css

```diff
@@ -360,11 +217,25 @@
 
 /* Second-and-more-level hovering. */
 #admin-bar .dropdown li li.expandable > a {
-  background: #45454A url(../images/arrow.png) no-repeat 145px center; /* LTR */
+  background: #45454A;
+}
+#admin-bar .dropdown li li.expandable > a::before {
+  content: '';
+  /*noinspection CssUnresolvedCustomProperty*/
+  mask-image: var(--icon-arrow-square-right); /* LTR */
+  mask-repeat: no-repeat;
+  background: #FFF;
+  width: 11px;
+  height: 11px;
```

Comment:
IMO 11px is awful also for the text (on large high-res screens absolutely unreadable). But that's unrelated here.

But if we made the dimensions based on `em` instead of `px` the icon would grow with the font, so if a theme or core somewhat later decided to switch to `rem` for admin-bar font-size...

Reply:
I've adjusted everything in admin_bar.css to be `em`, except for the base font size. Now you can scale the whole admin bar based on `#admin-bar { font-size: 16px }`, neat!